test/handler-tests.js

```diff
@@ -1005,6 +1007,43 @@ describe('handler', function() {
         .expectResult(() => mock.verify());
     });
 
+    it('should use the resolved value when "versionResolver" is provided', function() {
```

Comment:
Please, rename `describe` above to be just `version`, since we are testing two fields now